### PR TITLE
[AIRFLOW-6543][AIP-21] Promotion of contrib classes to the core

### PR DESCRIPTION
**Pull request by mik-laj**
More information: https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-21%3A+Changes+in+import+paths
https://docs.google.com/spreadsheets/d/1Z5jqsGz9QjmVDQAeI6NK8Aa-v6SQv8jfV8QzIZ4-q4w/edit#gid=0

---
Issue link: [AIRFLOW-6543](https://issues.apache.org/jira/browse/AIRFLOW-6543)

- [X] Description above provides context of the change
- [X] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/contrib/sensors/file_sensor.py [hunk 1]
```diff
@@ -16,54 +16,14 @@
...
-    @apply_defaults
-    def __init__(self,
-                 filepath,
-                 fs_conn_id='fs_default',
-                 *args,
-                 **kwargs):
-        super().__init__(*args, **kwargs)
-        self.filepath = filepath
-        self.fs_conn_id = fs_conn_id
+"""This module is deprecated. Please use `airflow.sensors.fs`."""
```
Comment (turbaszek) [inline airflow/contrib/sensors/file_sensor.py]: I am not sure if `fs` is a meaningful name, it's rather ambiguous.

Comment (mik-laj) [reply airflow/contrib/sensors/file_sensor.py]: It is a common abbreviation. We have HDFS already.  If you have suggestions, I invite you to a discussion on the mailing list.
https://lists.apache.org/thread.html/bdc1a070295121782e2e054ae62cac363ef84540b48f750916eae88a%40%3Cdev.airflow.apache.org%3E

Comment (potiuk) [reply airflow/contrib/sensors/file_sensor.py]: I agree `fs` is rather ambiguous. I missed that one when I looked at the spreadsheet. I would rather change it to `filesystem`. I don't think we need to discuss it in devlist if we agree during the PR that it's a better name. That's actually name that is still in the AIP-21 (thought it was in providers originally).

WDYT?

Comment (mik-laj) [reply airflow/contrib/sensors/file_sensor.py]: Ok. I will prepare PR.

**Review by potiuk — APPROVED** [review]

**Pull request merged**